src/Api/Controllers/CollectionsController.cs

```diff
@@ -135,7 +135,22 @@ public async Task<ListResponseModel<CollectionAccessDetailsResponseModel>> GetMa
     [HttpGet("")]
     public async Task<ListResponseModel<CollectionResponseModel>> Get(Guid orgId)
     {
-        IEnumerable<Collection> orgCollections = await _collectionService.GetOrganizationCollectionsAsync(orgId);
+        IEnumerable<Collection> orgCollections = null;
+        if (await _currentContext.ManageGroups(orgId))
+        {
+            // ManageGroups users need to see all collections to manage other users' collection access.
+            // This is not added to collectionService.GetOrganizationCollectionsAsync as that may have
+            // unintended consequences on other logic that also uses that method.
+            // This is a quick fix but it will be properly fixed by permission changes in Flexible Collections.
```

Comment:
If this is going to be fixed down the road, should we have an explicit task in our `Post Release` epic to properly track/update/remove this code? Or do we think an existing task encompasses the necessary clean-up?

Reply:
#3360 introduces a v2 version of this endpoint, so this code will be deleted when we remove the feature flags and clean up all the legacy code.